install-deps.sh

```diff
@@ -116,8 +116,15 @@ else
     case $ID in
     debian|ubuntu|devuan)
         echo "Using apt-get to install dependencies"
+        # Depdencies for Apache Arrow
+        curl https://dist.apache.org/repos/dist/dev/arrow/KEYS | sudo apt-key add -
+        $SUDO tee /etc/apt/sources.list.d/apache-arrow.list <<APT_LINE
+deb [arch=amd64] https://dl.bintray.com/apache/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/ $(lsb_release --codename --short) main
+deb-src https://dl.bintray.com/apache/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/ $(lsb_release --codename --short) main
+APT_LINE
+        $sudo apt-get update -y
         $SUDO apt-get install -y lsb-release devscripts equivs
-        $SUDO apt-get install -y dpkg-dev gcc
+        $SUDO apt-get install -y dpkg-dev gcc apt-transport-https curl gnupg libarrow-dev
```

Comment:
let's include the parquet libs here also please, so that we have another option to use arrow->parquet to write arrow to disk (into object) if possible.